k4MarlinWrapper/src/components/MarlinProcessorWrapper.cpp

```diff
@@ -155,8 +155,21 @@ StatusCode MarlinProcessorWrapper::initialize() {
     streamlog::out.init(std::cout, "k4MarlinWrapper");
     marlin::Global::parameters = new marlin::StringParameters();
     marlin::Global::parameters->add("AllowToModifyEvent", {"true"});
-    marlin::Global::parameters->add("RandomSeed", {"123456"});
-    // marlin::Global::EVENTSEEDER = new marlin::ProcessorEventSeeder() ;
+
+    // Random Service setup
+    std::vector<long> got_seeds{};
+    StatusCode        got_seeds_sc = randSvc()->engine()->seeds(got_seeds);
+    if (!got_seeds_sc.isSuccess())
+      warning() << "Random Service seeds could not be set" << endmsg;
+
+    auto sc_flatGen_init = m_flatGenerator.initialize(randSvc(), Rndm::Flat(0, 999999));
+    if (!sc_flatGen_init.isSuccess())
+      return Error("Cannot initialize Random Service based flat generator");
```

Comment:
Could this be removed, now that the seed is used directly?

Reply:
Yeah, tomorrow I will clean up the unused things. I would like to have this somewhere for documentation purposes, so maybe I will transfer it to some .md file.